mxcube3/routes/Utils.py

```diff
@@ -95,12 +97,142 @@ def my_execute_entry(self, entry):
         foo = ['collectOscillationFinished', 'collectOscillationFailed', 'warning']
         mxcube.collect.emit(random.choice(foo))
     elif isinstance(entry, qe.CharacterisationGroupQueueEntry):
+        logging.getLogger('HWR').info('[QUEUE] Executing CharacterisationGroupQueueEntry entry: %s' %entry)
```

Comment:
I'm wondering in general, without having thought to much about it, why we need this method my_execute_entry ? It as I understand it redefines the way entries are executed ?. 

I'm also wondering why there are some time.sleep calls, do we have some kind of synchronization problems ?